src/robusta/core/playbooks/internal/ai_integration.py

```diff
@@ -161,8 +161,9 @@ def holmes_workload_health(event: ExecutionBaseEvent, params: HolmesWorkloadHeal
             raise ActionException(ErrorCodes.HOLMES_UNEXPECTED_ERROR, "An unexpected error occured.")
 
 
-def build_conversation_title(params: HolmesConversationParams) -> str:
-    return f"{params.resource}, {params.ask} for issue {params.context.robusta_issue_id}"
+def build_conversation_title(params: HolmesConversationParams):
+    label_string = f"with alert labels {params.labels}" if params.labels else ""
```

Comment:
why did you remove the returned type annotation?

Also, I think the title with label is a little weird
It'll be:
`.... with labels with alert labels ...`
Is that on purpose?